client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/mapper/Taxi.kt

```diff
@@ -0,0 +1,35 @@
+package org.thechance.common.data.remote.mapper
+
+import androidx.compose.ui.graphics.Color
+import org.thechance.common.data.remote.model.TaxiDto
+import org.thechance.common.domain.entity.Taxi
+import org.thechance.common.domain.util.TaxiColor
+import org.thechance.common.domain.util.TaxiStatus
+
+fun TaxiDto.toEntity() = Taxi(
+    id = id,
+    plateNumber = plateNumber,
+    color = colorMapper[color] ?: Color.White,
+    type = type,
+    seats = seats,
+    status = statusMapper[status]?: TaxiStatus.ONLINE,
+    username = username,
+    trips = trips,
+)
+
+private val colorMapper: Map<Int, Color> = mapOf(
+    TaxiColor.RED.colorNumber to Color(0xFFF47373),
```

Comment:
It Should be better to get these colors from Local composition from our Theme , if not existed you shoud add them

Reply:
but this in data layer and mapping to domain layer , i can't use Local composition in here